layouts/partials/default.html

```diff
@@ -12,6 +12,11 @@
               {{ template "section-tree-nav" dict "sect" . "currentnode" $currentNode}}
               {{ end }}
             {{ end }}
+            {{ if .Content }}
+            <div>
+                <button class="btn btn-primary btn-block" id="genratePDF" > Get Pdf  </button>
```

Comment:
```suggestion
                <button class="btn btn-primary btn-block" id="generatePdf" > Get Pdf  </button>
```

Reply:
@jwflory please I don't get what I am supposed to do here.

Reply:
@Idadelveloper I used a `suggestion` feature in a Pull Request (remind me to show you how I did it!). You can choose to take my suggestions by committing them directly from the Pull Request. Alternatively, you can also fix it yourself and push the commit from your local repo too. 😀